docs/howto/hooks/webhooks.md

```diff
@@ -73,13 +73,15 @@ Upon execution, a webhook will send a request containing a JSON object with the
 | commit_message[^2]  | The message for the commit (or merge) that is taking place        | string |
 | committer[^2]       | Name of the committer                                             | string |
 | commit_metadata[^2] | The metadata for the commit that is taking place                  | string |
-| commit_id[^2,^4]    | The ID of the commit that is being created              | string |
+| commit_id[^2,^4]    | The ID of the commit that is being created                        | string |
 | tag_id[^3]          | The ID of the created/deleted tag                                 | string |
+| merge_source[^5]    | The source branch/tag/ref on merge operation                      | string |
```

Comment:
This renders as 4? (see image with purple arrow) 
![Screenshot 2025-02-27 at 11 38 12](https://github.com/user-attachments/assets/05b5570f-9423-4869-9bcf-1f1a365a5b5b)